release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1116,6 +1116,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 
 
+* Support level information about your cluster via an alert on the *Home* -> *Overview* -> *Details* card currently indicates the *Service Level Agreement (SLA)* is `unknown`. It will be fixed in a future version of {product-title}. For more information, see (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048059[*BZ#2048059]).
```

Comment:
Perhaps make "Currently, support level . . . " the introductory clause
I suggest dropping "via an alert" to improve clarity.
Maybe instead of "It" we can say "The incorrect status" 
You can remove the parentheses and formatting around the BZ#2048059 link.